arbnode/batch_poster.go

```diff
@@ -208,6 +212,84 @@ func NewBatchPoster(l1Reader *headerreader.HeaderReader, inbox *InboxTracker, st
 	return b, nil
 }
 
+// checkRevert checks blocks with number in range [from, to] whether they
+// contain reverted batch_poster transaction.
+func (b *BatchPoster) checkReverts(ctx context.Context, from, to *big.Int) (bool, error) {
+	if from.Cmp(to) == 1 {
```

Comment:
`arbmath` has some helper functions that can help clarify what Cmp checks are, e.g. `arbmath.BigGreaterThan` here (as-is it takes a bit of thinking to see that this is `if from > to`).

Reply:
Not relevant anymore since changed it to int64 types. Although I think using `Cmp` makes more sense anyway than writing wrapper for it.